geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapreduce/GeoMesaOutputFormat.scala

```diff
@@ -140,10 +144,15 @@ class GeoMesaRecordWriter(params: Map[String, String],
     val stats = statsCache.getOrElseUpdate(sftName, ds.stats.statUpdater(sft))
 
     val withFid = AccumuloFeatureWriter.featureWithFid(sft, value)
-    val encoder = encoderCache.getOrElseUpdate(sftName, SimpleFeatureSerializers(sft, ds.getFeatureEncoding(sft)))
+    val serializer = serializerCache.getOrElseUpdate(sftName,
```

Comment:
Do we want to extract this as utility function?  (And when?)

I'd be fine with a (yes, later) answer.  If so, then maybe we should add a ticket?